test/src/ariaqueryhandler.spec.ts

```diff
@@ -94,8 +98,10 @@ describeChromeOnly('AriaQueryHandler', () => {
       await page.setContent(
         '<div id="div"><button id="btn" role="button">Submit</button></div>'
       );
-      const button = await page.$('aria/[role="button"]');
-      const id = await button.evaluate((button: Element) => button.id);
+      const button = (await page.$('aria/[role="button"]'))!;
```

Comment:
should we assert instead of `!` in situations like this one? also `!` in `button!` on the line below seems redundant.

Reply:
There might be some redundancy because most of the changes were found through regular expressions. In particular, using `!` will guarantee that if it doesn't exist, it will break.